docs/kilnjs.md

```diff
@@ -84,6 +156,27 @@ module.exports = (schema) => {
   return schema;
 };
 ```
+You have access to all of the custom validators loaded in the kiln.validators object, so if you wish to write a reusable validation function that can be used across components, you could add a function to any of the existing validators or add a new one. For example you could use the same custom penelopecruz validator inline rather than adding it to the schema.validation object. However, running the validator this way will not add the error/warning to the health tab and you will need to communicate the error to the user manually using the snackbar component.
+
+```js
+const validators = schema.fieldName.validators(),
+  error = validators.penelopeCruz.kilnjsValidate(schema.title.value());
+```
+
+#### Form Validation
+
+Along with input level validation, you can also perform form validation using kiln.js.  To do this, you add a function to the schame named `formValidation`. This function is called just before a form is saved and if it returns `false` then the form will not be saved.  The snackBar element can be used to convey a message to the user explaining why the form wasn't saved.
+
+```js
+  schema.formValidation = () => {
```

Comment:
I think we should ident this block since its isolated code
```suggestion
schema.formValidation = () => {
  if (schema.size.value() === 'h2' && schema.title.value().length > 40) {
    kilnInput.showSnackBar({ message: 'When size is H2, the Title can\'t be longer than 40 chars', position: 'center' })
    return false;
  }

  return true;
};
```

Reply:
👍